README.md

```diff
@@ -30,11 +30,67 @@
 * The API itself is really simple. You're task is to peek at the endpoints found in the `server.js` file and write docs for each one, then write the tests for the end points.
 
 ## DOCUMENTATION GOES HERE
+# Server Testing - Games API Documentation
+This API was created in order to help understand Server testing using ```Chai```, ```Chai-HTTP``` and ```Sinon```.
 
+The API contains routes that use the following HTTP verbs: POST, GET, PUT and DELETE.
+
+It will allow users to view a list of games with their genre dateReleased and title. Users will also be able to create new game entries to the database, edit a given game and delete a given game.
```

Comment:
This is very nice. A high level overview of the application is very meaningful in API documentation.  As I am sure you have seen, documentation is not always written with a non-expert in mind. Any kind of plain English you can provide as an overview for the whole project, or the individual aspects is not only useful to you as you develop, but to any software developers or end-users that will work with your project after you are done with it.